t/sanity.t

```diff
@@ -21,6 +21,7 @@ our $HttpConfig = qq{
 
 no_long_string();
 #no_diff();
+no_shuffle();
```

Comment:
Is this required?

Reply:
We'd better add it so that successive related similar tests are tested together.